benchmark/single-source/LuhnAlgoEager.swift

```diff
@@ -144,7 +144,7 @@ let charToString = { (c: Character) -> String in String(c) }
 let charToInt = stringToInt • charToString
 
 func sum<S: Sequence>(_ nums: S)->S.Iterator.Element where S.Iterator.Element: FixedWidthInteger {
-    return nums.reduce(0) { $0.0 + $0.1 }
+    return nums.reduce(0, +)
```

Comment:
That'll start failing eventually.. I think. When the SE-110 is reconsidered again.

Reply:
I dislike that change so this is a pro not a con AFAIAC :), but the reason I changed it is cos it's now `$0` and `$1` so I _think_ it's SE-110-compatible.